pulp_container/app/registry_api.py

```diff
@@ -647,7 +647,7 @@ def put(self, request, path, pk=None):
             try:
                 task = Task.objects.filter(
                     name__endswith="add_and_remove",
-                    reserved_resources_record__resource=f"upload:{pk}",
+                    reserved_resources_record=[f"upload:{pk}"],
```

Comment:
Good catch! This is actually a bugfix, right?

Reply:
yeah, this was forgotten to be updated to reflect new tasking system changes